server

```diff
@@ -91,7 +91,7 @@ echo_msg() {
 run_cmd() {
   CMD=("$@")
   if [ $DRY_RUN -eq 0 ]; then
-    "${CMD[@]}"
+    eval "${CMD[@]}"
```

Comment:
Why this is needed?

Reply:
`eval` is used to joined arguments in to a string. I used it where the installation command string includes few "=" and "}" otherwise it spit it to few strings. I tried with \ but it was not work for me.